### PR TITLE
#1254 - Add support for JSONSchema constants

### PR DESCRIPTION
**Pull request by Lily418**
The JSON reference for constant values in JSONSchema says
"It should be noted that const is merely syntactic sugar for an enum
with a single element"
https://json-schema.org/understanding-json-schema/reference/generic.html#constant-values

So this is the way I've implemented them.

### CODE REVIEW

File: packages/examples/src/1254.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  },
+  required: ['version']
+};
+
+export const uischema = {
+  type: 'HorizontalLayout',
+  elements: [
+    {
+      type: 'Control',
+      label: 'Value',
```
Comment (edgarmueller) [inline packages/examples/src/1254.ts]: Just a minor that I noticed, this should be probably be labelled as 'Name'.

**Comment by Lily418** [comment]
@edgarmueller have updated ✨

File: packages/core/test/testers.test.ts [hunk 2]
```diff
@@ -515,6 +515,13 @@ test('test isEnumControl', t => {
       properties: { foo: { type: 'number', enum: [1, 2] } }
     })
   );
+  t.true(
+    isEnumControl(t.context.uischema, {
+      type: 'object',
+      properties: { foo: { const: '1.0' } },
+      required: ['version']
```
Comment (eneufeld) [inline packages/core/test/testers.test.ts]: the property `version` does not exist in this schema, I think this is a left over from the example, I would suggest to remove it

**Comment by Lily418** [comment]
@eneufeld thanks, I've removed that now

**Pull request merged**